### DISCUSSION TITLE
No issue but some questions....

### DISCUSSION THREAD

**Discussion by WhiteLionATX** [phoddie/node-red-mcu#20] [Q&A]
I saw this project on YT (Node Red Con 2022).
Now I ve some questions:
1. where is the place to get help / started with this ? 
2. I currently use ESPs + C++ with the libs for sensors I use. How can my "Node Red ESP" get data from sensors ? How are the libs includes ? Or did I get something totally wrong with this project ?  
3. Thx :)

**Comment by phoddie** [comment]
Welcome. Thanks for watching my presentation.

> where is the place to get help / started with this ?

You are in the right place!

> I currently use ESPs + C++ with the libs for sensors I use. How can my "Node Red ESP" get data from sensors ? 

There are several ways. The best place to start is by using an existing sensors driver. [There are many](https://github.com/Moddable-OpenSource/moddable/tree/public/modules/drivers/sensors) in the Moddable SDK that can be used with the Sensor Node that is built-into the Node-RED MCU Edition.

> How are the libs includes ?

The Sensor Node's [readme.md](https://github.com/phoddie/node-red-mcu/tree/main/nodes/sensor/readme.md) has an example of adding a sensor driver to the project and a basic Node-RED flow that uses it.

Of course, you can also write your own sensor drivers -- in JavaScript or C/C++ -- but starting with an existing driver that already works is easier and a great way to learn.

(FYI – since this isn't an Issue, I'm going to convert it to a discussion)
